### PR TITLE
fix: add width to linear progress wrapper

### PR DESCRIPTION
**Pull request by jenseo**
making sure the child of linear progress always have space to calculate percentage from

### CODE REVIEW

File: src/components/charts/LinearProgress/LinearProgress.tsx [hunk 1]
```diff
@@ -47,6 +47,8 @@ const Wrapper = styled.div`
   height: ${({ theme }) => theme.spacer}px;
   border-radius: ${({ theme }) => theme.spacer}px;
   position: relative;
+  /* Width needed to make sure the child has space to use */
```
Comment (filiptammergard) [inline src/components/charts/LinearProgress/LinearProgress.tsx:50]: Hmm, isn't the case more like "Width needed to make sure component takes up full width in flex containers"? 🙂

**Review by filiptammergard — CHANGES_REQUESTED** [review]
Can you add this for StepProgress as well? Should have the same problem!

File: src/components/charts/LinearProgress/LinearProgress.tsx (hunk 1 above)
Comment (jenseo) [reply src/components/charts/LinearProgress/LinearProgress.tsx:50]: Yeah, that might be a better description perhaps! Thanks, will change!

**Comment by jenseo** [comment]
> 

That's true, I actually thought it would take up the whole space since the Wrapper in the step progress is display flex, but that was a snedtänk from my side. Tried it out and you're definitely correct :)

**Review by filiptammergard — APPROVED** [review]

**Pull request merged**